<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Proyecto</class>
 <widget class="QDialog" name="Proyecto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>472</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Proyectos</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/imagenes/graph.ico</normaloff>:/imagenes/graph.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{	
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(232, 239, 252, 255), stop:1 rgba(241, 248, 239, 255));
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QStackedWidget" name="sWPanelCentral">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="5" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Fecha Termino:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_nuevo">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {	
    padding: 10px 20px 10px 20px;
}
</string>
             </property>
             <property name="text">
              <string>NUEVO PROYECTO</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="nombreProyecto">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maxLength">
              <number>4500</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Nombre: </string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Fecha Inicio:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QDateEdit" name="fechaIP">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Descripci√≥n:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QPlainTextEdit" name="InfoProyecto">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="cursor" stdset="0">
              <cursorShape>IBeamCursor</cursorShape>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ingresa una descripcion del proyecto que no supere los 4500 caracteres.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="plainText">
              <string/>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="btnAgregar">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>90</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>-1</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Insertar los datos?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {	
	background-color: rgb(0, 158, 5);
	border-radius: 2px;
    font-family: Arial;
    color: #ffffff;
    font-size: 15px;
    padding: 5px 5px 5px 5px;

}

QPushButton:pressed{ 
    color: #000000;
	border: 2px solid rgb(15, 103, 0);
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(5, 0, 9, 118));
}

QPushButton:hover {
  background-color: qlineargradient(spread:pad, x1:0.489, y1:1, x2:0.494, y2:0.0113636, stop:0 rgba(0, 117, 0, 255), stop:1 rgba(0, 255, 0, 255));	  
  text-decoration: none;
}</string>
               </property>
               <property name="text">
                <string>Agregar</string>
               </property>
               <property name="icon">
                <iconset resource="images.qrc">
                 <normaloff>:/imagenes/Word-Processor.ico</normaloff>:/imagenes/Word-Processor.ico</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnCerrar">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>70</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>-1</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cierra esta ventana&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {	
	background-color: rgb(0, 158, 5);
	border-radius: 2px;
    font-family: Arial;
    color: #ffffff;
    font-size: 15px;
    padding: 5px 5px 5px 5px;

}

QPushButton:pressed{ 
    color: #000000;
	border: 2px solid rgb(15, 103, 0);
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(5, 0, 9, 118));
}

QPushButton:hover {
  background-color: qlineargradient(spread:pad, x1:0.489, y1:1, x2:0.494, y2:0.0113636, stop:0 rgba(0, 117, 0, 255), stop:1 rgba(0, 255, 0, 255));	  
  text-decoration: none;
}</string>
               </property>
               <property name="text">
                <string>Cerrar</string>
               </property>
               <property name="icon">
                <iconset resource="images.qrc">
                 <normaloff>:/imagenes/Xion.ico</normaloff>:/imagenes/Xion.ico</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnVer">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>-1</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ve los proyectos actuales?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {	
	background-color: rgb(0, 158, 5);
	border-radius: 2px;
    font-family: Arial;
    color: #ffffff;
    font-size: 15px;
    padding: 5px 5px 5px 5px;

}

QPushButton:pressed{ 
    color: #000000;
	border: 2px solid rgb(15, 103, 0);
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(5, 0, 9, 118));
}

QPushButton:hover {
  background-color: qlineargradient(spread:pad, x1:0.489, y1:1, x2:0.494, y2:0.0113636, stop:0 rgba(0, 117, 0, 255), stop:1 rgba(0, 255, 0, 255));	  
  text-decoration: none;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="images.qrc">
                 <normaloff>:/imagenes/Media-Player.ico</normaloff>:/imagenes/Media-Player.ico</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="6" column="0">
            <widget class="QDateEdit" name="fechaTP">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Buscar:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="5">
            <widget class="QLineEdit" name="txtBuscar">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="placeholderText">
              <string>Ingrese Nombre del proyecto</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QToolButton" name="btnVolver">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Volver a la ventana de craci√≥n de proyectos?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton {	
	background-color: rgb(0, 158, 5);
	border-radius: 2px;
    font-family: Arial;
    color: #ffffff;
    font-size: 15px;
    padding: 5px 5px 5px 5px;

}

QToolButton:pressed{ 
    color: #000000;
	border: 2px solid rgb(15, 103, 0);
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(5, 0, 9, 118));
}

QToolButton:hover {
  background-color: qlineargradient(spread:pad, x1:0.489, y1:1, x2:0.494, y2:0.0113636, stop:0 rgba(0, 117, 0, 255), stop:1 rgba(0, 255, 0, 255));	  
  text-decoration: none;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="images.qrc">
               <normaloff>:/imagenes/flechaIzq.ico</normaloff>:/imagenes/flechaIzq.ico</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="6">
            <widget class="QLabel" name="label_listado">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {	
    padding: 10px 20px 10px 20px;
}
</string>
             </property>
             <property name="text">
              <string>LISTADO DE PROYECTOS ACTUALES</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="6">
            <widget class="QTableView" name="tablaProyectos">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="4" column="5">
            <widget class="QPushButton" name="btnEliminar">
             <property name="styleSheet">
              <string notr="true">QPushButton {	
	background-color: rgb(0, 158, 5);
	border-radius: 2px;
    font-family: Arial;
    color: #ffffff;
    font-size: 15px;
    padding: 5px 5px 5px 5px;

}

QPushButton:pressed{ 
    color: #000000;
	border: 2px solid rgb(15, 103, 0);
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(5, 0, 9, 118));
}

QPushButton:hover {
  background-color: qlineargradient(spread:pad, x1:0.489, y1:1, x2:0.494, y2:0.0113636, stop:0 rgba(0, 117, 0, 255), stop:1 rgba(0, 255, 0, 255));	  
  text-decoration: none;
}</string>
             </property>
             <property name="text">
              <string>Eliminar</string>
             </property>
             <property name="icon">
              <iconset resource="images.qrc">
               <normaloff>:/imagenes/Xion.ico</normaloff>:/imagenes/Xion.ico</iconset>
             </property>
            </widget>
           </item>
           <item row="4" column="4">
            <widget class="QPushButton" name="btnOpen">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Abrir el proyecto seleccionado?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {	
	background-color: rgb(0, 158, 5);
	border-radius: 2px;
    font-family: Arial;
    color: #ffffff;
    font-size: 15px;
    padding: 5px 5px 5px 5px;

}

QPushButton:pressed{ 
    color: #000000;
	border: 2px solid rgb(15, 103, 0);
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(5, 0, 9, 118));
}

QPushButton:hover {
  background-color: qlineargradient(spread:pad, x1:0.489, y1:1, x2:0.494, y2:0.0113636, stop:0 rgba(0, 117, 0, 255), stop:1 rgba(0, 255, 0, 255));	  
  text-decoration: none;
}</string>
             </property>
             <property name="text">
              <string>Abrir</string>
             </property>
             <property name="icon">
              <iconset resource="images.qrc">
               <normaloff>:/imagenes/Power.ico</normaloff>:/imagenes/Power.ico</iconset>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="3">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nombreProyecto</tabstop>
  <tabstop>fechaIP</tabstop>
  <tabstop>fechaTP</tabstop>
  <tabstop>InfoProyecto</tabstop>
  <tabstop>btnAgregar</tabstop>
  <tabstop>btnCerrar</tabstop>
  <tabstop>btnVer</tabstop>
  <tabstop>txtBuscar</tabstop>
  <tabstop>tablaProyectos</tabstop>
  <tabstop>btnVolver</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCerrar</sender>
   <signal>clicked()</signal>
   <receiver>Proyecto</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
